src/protocol.py

```diff
@@ -168,13 +168,14 @@ def checkSocksIP(host):
 
 def isProofOfWorkSufficient(data,
                             nonceTrialsPerByte=0,
-                            payloadLengthExtraBytes=0):
+                            payloadLengthExtraBytes=0,
+                            recvTime=0):
```

Comment:
Could you add a parameter type definition in the docstring please? It's not obvious what kind of time object you want to be passed in. I believe you're expecting a float representing unix time rather than a python time/datetime type.

Reply:
Good idea, @f97ada87 please add docstring.

Reply:
Docstring added; also an integer cast to match the treatment given to time.time()